README.md

```diff
@@ -1,4 +1,5 @@
 [![Build Status](https://dev.azure.com/azure/template-analyzer/_apis/build/status/BuildAndTest?branchName=main)](https://dev.azure.com/azure/template-analyzer/_build/latest?definitionId=91&branchName=main)
+[![Code Coverage](https://shields.io/azure-devops/coverage/azure/template-analyzer/91)](https://dev.azure.com/azure/template-analyzer/_build/latest?definitionId=91&branchName=main)
```

Comment:
I'm noticing the "rich diff" view doesn't display this correctly for some reason.  If you just click on the branch name above to view the code in this branch, it shows correctly on the readme.

Reply:
I also couldn't figure out any way to link directly to the test results view of the latest build.  I decided to just link to the build in lieu of a better option.